intel-sgx/enclave-runner/src/usercalls/mod.rs

```diff
@@ -49,10 +49,11 @@ lazy_static! {
     static ref DEBUGGER_TOGGLE_SYNC: Mutex<()> = Mutex::new(());
 }
 
-const EV_ABORT: u64 = 0b0000_0000_0000_1000;
+const EV_ABORT: u64 = 0b0000_0000_0001_0000;
```

Comment:
`EV_ABORT` is not an event in the sense that it could be passed to `send()` or `wait()` usercalls in enclave code. However, it's easier for the enclave runner implementation to lump it in with events. Also note that this constant is not public (it's an implementation detail of enclave-runner).

Reply:
Sure, but could you add this explanation as a comment?